services/event_bus/event_handler.go

```diff
@@ -79,7 +79,12 @@ func handleHAPUpdate(conf *configuration.Configuration, conn *zk.Conn) bool {
 		log.Panicf("Cannot read template file: %s", err)
 	}
 
-	templateData := haproxy.GetTemplateData(conf, conn)
+	templateData, err := haproxy.GetTemplateData(conf, conn)
+	
+	if err != nil {
+	  log.Printf("Not updating haproxy because we failed to retrieve template data: \n %s\n", err)
+	  return false
```

Comment:
I will implement a delayed reload event to ensure a reload do happen when template data is extracted correctly.